core/Cargo.toml

```diff
@@ -209,6 +209,7 @@ reqwest = { version = "0.11.13", features = [
 rocksdb = { version = "0.20.1", default-features = false, optional = true }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
+sha2 = "0.10"
```

Comment:
Please add this as optional and only enable in `services-azblob`.

Reply:
Done